api-tools/src/lld-holders-report.ts

```diff
@@ -0,0 +1,136 @@
+	"5EYCAe5iXF2Ya2c2iKjeFWUAXAEcMyoKCBPvhRy8YprHTLNd":
+		"Metaverse Land Registry Office",
+	"5EYCAe5iXF2YZfPZ4arKGSYZvQruDacGHkuw4qAsQSQsKpMK": "Asset Registry Office",
+	"5EYCAe5iXF2YZpCZr7ALYUUYaNpMXde3NUXxYn1Sc1YRM4gV":
+		"Ministry of Finance Office",
+	"5FBQRNJfzsttYvw1XnSwxwSUmb7A3EYm4q8aiscADREqvzYz":
+		"Wrapped LLD Token Contract",
+	"5GsBCWqN6mnrq4arSMBuT4uQ8GJKwgeBsa5UyCvTN6DyVj3S": "Emirex",
+};
+
+const formatBalance = (v: BN) => v.div(new BN(1000000000000)).toString();
+
+async function main() {
+	const wsProvider = new WsProvider("wss://mainnet.liberland.org");
+	const api = await ApiPromise.create({ provider: wsProvider });
+
+	const data: any = {};
+
+	const accounts = await api.query.system.account.entries();
+	accounts.forEach(([{ args: key }, acc]) => {
+		const address = key.toString();
+		const details = (acc as any).data;
+		const liquid_lld_balance = formatBalance(
+			details.free.sub(details.frozen)
+		);
+		const frozen_lld_balance = formatBalance(details.frozen);
+		const reserved_lld_balance = formatBalance(details.reserved);
+		const total_lld_balance = formatBalance(
+			details.free.add(details.reserved)
+		);
+		data[address] = {
+			liquid_lld_balance,
+			frozen_lld_balance,
+			reserved_lld_balance,
+			total_lld_balance,
+			address,
+		};
+	});
+
+	const identities = await api.query.identity.identityOf.entries();
+	identities.forEach(([{ args: key }, acc]) => {
+		const address = key.toString();
+		const raw = (acc as any).toJSON().info.display.raw ?? "0x";
+		const display = Buffer.from(raw.substring(2), "hex").toString("utf-8");
+		if (data[address] === undefined) data[address] = { address };
+		const id = (acc as any).unwrap();
+		data[address].display = display;
+		data[address].identity = id.info.toHuman();
+		data[address].is_citizen =
+			id.info.additional.some(
+				([k, v]: [any, any]) => k.asRaw.eq("citizen") && v.asRaw.eq("1")
+			) &&
+			id.judgements.some(
+				([registrar, judgement]: [any, any]) =>
+					registrar.eq(0) && judgement.isKnownGood
+			);
+		data[address].is_eresident =
+			id.info.additional.some(
+				([k, v]: [any, any]) =>
+					k.asRaw.eq("eresident") && v.asRaw.eq("1")
+			) &&
+			id.judgements.some(
+				([registrar, judgement]: [any, any]) =>
+					registrar.eq(0) && judgement.isKnownGood
+			);
+	});
+
+	const llmLiquid = await api.query.assets.account.entries(1);
+	llmLiquid.forEach(([{ args: key }, value]: [any, any]) => {
+		const address = key[1].toString();
+		if (value.isSome) {
+			if (data[address] === undefined) data[address] = { address };
+			data[address].liquid_llm_balance = formatBalance(
+				value.unwrap().balance
+			);
+		}
+	});
+
+	const llmStaked = await api.query.llm.llmPolitics.entries();
+	llmStaked.forEach(([{ args: key }, value]: [any, any]) => {
+		const address = key[0].toString();
+		if (data[address] === undefined) data[address] = { address };
+		data[address].staked_llm_balance = formatBalance(value);
+	});
+
+	console.log(
```

Comment:
console.log

Reply:
What about it? It outputs the header to stdout, so that you can redirect it to CSV file and load it in Excel or whatever.

Reply:
In other words: it's there on purpose, it's not a debugging leftover.